src/components/MultiAutocompleteSelectField/MultiAutocompleteSelectField.tsx

```diff
@@ -205,28 +207,35 @@ const MultiAutocompleteSelectFieldComponent: React.FC<MultiAutocompleteSelectFie
                     disabled={disabled}
                   />
                   {isOpen && (
-                    <MultiAutocompleteSelectFieldContent
-                      add={
-                        add && {
-                          ...add,
-                          onClick: () => {
-                            add.onClick();
-                            closeMenu();
+                    <Popper
+                      anchorEl={anchor.current}
+                      open={isOpen}
+                      style={{ width: anchor.current.clientWidth, zIndex: 10 }}
```

Comment:
Do we need explicit `zIndex`, isn't the default enough?

Reply:
For some reason some elements, eg. input labels have higher zIndex, so I raised it here as MUI is not able to do it globally using theme.